styles/library.scss

```diff
@@ -0,0 +1,174 @@
+/**
+ * This stylesheet contains styles for pattern-library components.
+ */
+@use 'variables' as var;
+@use './mixins/layout';
+@use './mixins/atoms';
+@use './mixins/patterns/containers';
+
+.Library {
+  &__heading2 {
+    border-left: 6px solid var.$color-brand;
+    padding-left: 0.5em;
+  }
+
+  &__heading3 {
+    font-weight: normal;
+    font-style: italic;
+  }
+
+  &__heading4 {
+    font-style: italic;
+  }
```

Comment:
I'm not married to this naming convention going forward; it's an interim step of isolating these styles.